### ISSUE TITLE
Experiments of linear classification on different numbers of GPUs

### ISSUE THREAD

**Issue by yxgeee** [facebookresearch/swav#20]
Hello,

Thanks for your inspiring paper and code.

I trained SwAV with a batch size of 4096 for 200 epochs and then trained a linear classifier with your default setting (batch size of 256 on 8 GPUs), achieving 74.5% top-1 accuracies. I wanted to fasten the linear classifier training process, so I tried to train it with a batch size of 2048 on 64 GPUs and left all the other settings the same. I observed 73.3% in terms of top-1, showing a slight drop from your default setting. 

So I am wondering how to train the linear classifier on 64 GPUs and achieve similar performance as training on 8 GPUs, e.g. tuning some hyper-parameters? Looking forward to your reply. 

Thanks

**Comment by yxgeee** [comment]
I have solved this issue by tuning the learning rate.

**Comment by mathildecaron31** [comment]
Great, feel free to re-open the issue if needed
